docs/metrics.md

```diff
@@ -11,3 +11,13 @@ The Managed Upgrade Operator reports the following metrics:
 - `upgrade_worker_start`: The start time of the worker upgrades
 - `upgrade_worker_completion`: The completion time of the worker upgrades
 - `upgrade_complete`: The completion time of the managed upgrade
+
+## Metrics use to alert SRE
```

Comment:
This section _may_ end up changing in the very near future when OSD-6778 is finished. But it's fine for now.